### ISSUE TITLE
Long delay on start seems to come from BakingTray

### ISSUE THREAD

**Issue by raacampbell** [SainsburyWellcomeCentre/BakingTray#384]
I tried running an acquisition with just one section on FCI BS and it started right away. 200 sections is slow. This leads me to suspect that I'm somewhere preallocating a large array or doing something else similar before starting the acquisition and that is the reason for the long delay in the start. It can't be ScanImage. So look in the `bake` routine and see what is taking up the time.

**Comment by raacampbell** [comment]
I wasn't expecting that... Problem is in `BT.checkIfAcquisitionIsPossible`

![image](https://user-images.githubusercontent.com/5427811/122203336-d8579700-ce95-11eb-8f77-dba8d73bf4b5.png)

**Comment by raacampbell** [comment]
The slowness is because:

```
./+BakingTray/+gui/@acquisition_view/setupListeners.m:    obj.listeners{end+1}=addlistener(obj.model, 'currentSectionNumber', 'PostSet', @obj.updateStatusText)
```

**Comment by raacampbell** [comment]
Should be fixed now: e8b87d186a54c0ebd4fb7b7cbac6b22e1cedc201

**Comment by raacampbell** [comment]
I confirm this is fixed
